### DISCUSSION TITLE
Does termuisic have an icon?

### DISCUSSION THREAD

**Discussion by plum** [tramhao/termusic#305] [General]
Termusic desktop icon:
I created a desktop file for termuisc, and woant to put an icon into that. I wonder, does termusic have it's own icon (or intend to), or shoul ld go ahead and concoct my own, if you don't intend to have one?

**Comment by tramhao** [comment]
Under assets folder there is a png file, for displaying as an icon in discord, showing what's playing right now. If you have better icon, we'd love to incorporate. Thanks.

**Reply by GaugeAndGravity** [reply]
I'm trying to make one. How does this look? It's a stupidly simple idea: terminal and music...
<img width="1124" height="1114" alt="image" src="https://github.com/user-attachments/assets/028c509e-d17a-42f1-9421-e1a90da45ca3" />
